### PR TITLE
Adds PENLITE holobarriers

### PR DESCRIPTION
**Pull request by ExcessiveUseOfCobblestone**
:cl: Cobby (based off Wesoda25's idea)
add: Adds the PENLITE holobarrier. A holographic barrier which halts individuals with bad diseases!
/:cl:

Adds a simple way to quarantine individuals but doesn't make it roundstart, as it has potential to make 2 antags have a very difficult time: Bad virologists and (mainly) sentient diseases. Uses a sliver of silver because of this.

It uses the exact same proc as the MEDHUD, so it CAN be countered by **buffing stealth**. Virologists can also **activate the free-access** so long as they do it while not infected with anything bad.

That said, it has a **build delay** and is **relatively easy to destroy**. Think of it as a **more obvious visual for people without huds to know "hey, I should probably avoid everyone else until I get better" and not necessarily an actual barrier device.**

sprites edited by me if you like them, by someone else if you don't!

### CODE REVIEW

**Comment by Militaires** [comment]
show the sprites dickbiglet............................................................................. also does this stop you from contracting diseases or from spreading diseases? :eyes:

**Comment by ExcessiveUseOfCobblestone** [comment]
neither, it solely limits you to a certain area ergo stopping spread.

also look at icon diff bot fool

**Comment by optimumtact** [comment]
this is dumb and doesn't even make any sense lore wise

**Comment by Tlaltecuhtli** [comment]
time to make 1x1 traps  for regen coma nerds

**Comment by granpawalton** [comment]
id be all for it if either it was restricted to viro or CMO or if it was run only denying like sec holos

**Comment by ExcessiveUseOfCobblestone** [comment]
These take 2-3 hits and they’re torn down, they’re not actual barriers for the purpose of denying entry to someone who wants it. Very easily escapable (they’re holographic)! You can also trip them by simply clicking them before infection.

They also do not spot diseases your medhuds can’t pick up.

I’m not entertaining the coma comment, its a coma they can’t do much to begin with.

Reminder we also have holographics that block people and air, this just scans. I don’t find that out of place. These have a build delay as well so they can’t be spammed.

**Comment by ExcessiveUseOfCobblestone** [comment]
That said, I think I'll set the normal one to 1 barrier per and a longer creation time to reassure no spam is done.

**Comment by MrDoomBringer** [comment]
@ExcessiveUseOfCobblestone Three per barrier would be better imo.

**Comment by Arianya** [comment]
Three per is more in line with other holobarrier dispensers yeah.

**Comment by ExcessiveUseOfCobblestone** [comment]
Hmmm yeah, just trying to dance with numbers until everyone is happy (see: never :^] )

File: code/modules/research/designs/medical_designs.dm [hunk 1]
```diff
@@ -595,3 +595,13 @@
 	build_path = /obj/item/disk/surgery/necrotic_revival
 	category = list("Medical Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_MEDICAL | DEPARTMENTAL_FLAG_SCIENCE
+
+/datum/design/holobarrier_med
+	name = "PENLITE holobarrier projector"
+	desc = "PENLITE holobarriers, a device that halts individuals with malicious diseases."
+	build_type = PROTOLATHE
+	build_path = /obj/item/holosign_creator/medical
+	materials = list(MAT_METAL = 500, MAT_GLASS = 500, MAT_SILVER = 100) //a hint of silver since it can troll 2 antags (bad viros and sentient disease)
```
Comment (vuonojenmustaturska) [inline code/modules/research/designs/medical_designs.dm]: silver is a roundstart material on all stations but maybe omega?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/research/designs/medical_designs.dm]: It's a deterrence compounded with the actual roadblock of techwebs.

**Pull request merged**